topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -0,0 +1,290 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# IntelliJ Platform Gradle Plugin 2.x (EAP)
+
+<link-summary>IntelliJ Platform Gradle Plugin user and migration guide.</link-summary>
+
+<tldr>
+
+**Current Release**: Early Access Preview
+
+**GitHub**: [Releases & Changelog](https://github.com/JetBrains/gradle-intellij-plugin/releases), [Issue Tracker](https://github.com/JetBrains/gradle-intellij-plugin/issues)
+
+**Slack**: [#intellij-platform-gradle-plugin](https://jetbrains-platform.slack.com/archives/C05C80200LS) on the [JetBrains Platform Slack](https://plugins.jetbrains.com/slack/)
+
+</tldr>
+
+The _IntelliJ Platform Gradle Plugin 2.x_ is a plugin for the Gradle build system to help configure your environment for building, testing, verifying, and publishing plugins for IntelliJ-based IDEs.
+
+It is going to replace the current _[](tools_gradle_intellij_plugin.md) (1.x)_ in the future.
+
+> This plugin is currently in **Early Access** and may not support all features and project setups yet (see also [](#requirements)).
+> Please report bugs or problems in the above-linked issue tracker or Slack channel.
+>
+> Any documentation issues on this page should be reported using the feedback form on the bottom of this page.
+>
+> Thanks a lot in advance for your feedback!
+>
+{title="Early Access Status" style="warning"}
+
+## Requirements
+
+The following platforms and environments are supported:
+
+- IntelliJ Platform: 2022.3 and later
+- Java Runtime: 17 and later
+- Gradle: 8.1 and later
+
+## Usage
+
+To use the current Early Access Preview snapshot versions, add to your <path>settings.gradle.kts</path> file:
+
+```kotlin
+pluginManagement {
+  repositories {
+    maven("https://oss.sonatype.org/content/repositories/snapshots/")
+    gradlePluginPortal()
+  }
+}
+```
+
+In the <path>build.gradle.kts</path> file, replace the existing reference to the [](tools_gradle_intellij_plugin.md) plugin (`org.jetbrains.intellij`) with:
+
+```kotlin
+plugins {
+  id("org.jetbrains.intellij.platform") version "%intellij-platform-gradle-plugin-version%"
+}
+```
+
+### Available Subplugins
+
+The plugin has a new ID: `org.jetbrains.intellij.platform`, but also was split into subplugins.
```

Comment:
Wdyt about putting this paragraph above the "Available Subplugins" section like this: Please note that the plugin has a new ID `org.jetbrain.intellij.platform`. In Addition, the plugin was split into subplugins that can be applied separately. This allows for applying only a subset of features, e.g. when you only...

Then I would rename the heading "Available Subplugins" to just "Subplugins".

Reply:
+1